components/infobox/wikis/starcraft2/infobox_person_player.lua

```diff
@@ -236,82 +240,78 @@ function StarCraft2Player._getLPDBrecursive(cond, query, queryType)
 	return data
 end
 
-function StarCraft2Player._getEarningsMedalsData(player)
+function CustomPlayer._getEarningsMedalsData(player)
 	local cond = '[[date::!1970-01-01 00:00:00]] AND ([[prizemoney::>0]] OR ' ..
 		'([[mode::1v1]] AND ([[placement::1]] OR [[placement::2]] OR [[placement::3]]' ..
 		' OR [[placement::4]] OR [[placement::3-4]]))) AND ' ..
 		'[[participant::' .. player .. ']]'
 	local query = 'liquipediatier, liquipediatiertype, placement, date, prizemoney, mode'
 
-	local data = StarCraft2Player._getLPDBrecursive(cond, query, 'placement')
+	local data = CustomPlayer._getLPDBrecursive(cond, query, 'placement')
 
 	local earnings = {}
-	local Medals = {}
+	local medals = {}
 	earnings['total'] = {}
-	Medals['total'] = {}
+	medals['total'] = {}
 	local earnings_total = 0
 
 	if type(data[1]) == 'table' then
-		for i = 1, #data do
+		for i=1,#data do
```

Comment:
```suggestion
		for i = 1, #data do
```

Reply:
Change to `in pairs` or `in ipairs` iterator instead of an index iteration.